examples/morpheusvm/cmd/morpheus-cli/cmd/deploy.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/ava-labs/avalanchego/tests/fixture/tmpnet"
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/consts"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/rpc"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/tests/workload"
+	"github.com/ava-labs/hypersdk/tests/fixture"
+	"github.com/ava-labs/hypersdk/utils"
+)
+
+const (
+	owner      = "morpheus-cli"
+	devNetPort = "9560"
+)
+
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Quickly deploy an instance of MorpheusVM",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		genesis, _, err := workload.New(minBlockGap)
+		if err != nil {
+			return err
+		}
+		genBytes, err := json.Marshal(genesis)
+		if err != nil {
+			return err
+		}
+		nodes := tmpnet.NewNodesOrPanic(numOfNodes)
+		// Set static port for DevNet
+		nodes[0].Flags["http-port"] = devNetPort
+		subnet := fixture.NewHyperVMSubnet(
+			consts.Name,
+			consts.ID,
+			genBytes,
+			nodes...,
+		)
+
+		timeOut := 2 * time.Minute
+
+		ctx, cancel := context.WithTimeout(context.Background(), timeOut)
+		defer cancel()
+
+		network := fixture.NewTmpnetNetwork(owner, nodes, subnet)
+		if err := tmpnet.BootstrapNewNetwork(
+			ctx,
+			os.Stdout,
+			network,
+			"",
+			avalancheGoPath,
+			avalancheGoPluginDir,
+		); err != nil {
+			utils.Outf(err.Error())
+			return err
+		}
```

Comment:
If we need to port this code from the e2e tests, can we wrap the construction into a single function? Similar comment here https://github.com/ava-labs/hypersdk/pull/1320#discussion_r1721813693